helm-charts/splunk-otel-collector/templates/_helpers.tpl

```diff
@@ -324,6 +324,20 @@ compatibility with the old config group name: "otelAgent".
 {{- end }}
 {{- end -}}
 
+{{/*
+The name of the gateway service.
+*/}}
+{{- define "splunk-otel-collector.gatewayServiceName" -}}
+{{  (include "splunk-otel-collector.fullname" . ) | trunc 63 | trimSuffix "-" }}
+{{- end -}}
+
+{{/*
+Whether the gateway is enabled, either through network explorer, or through its own flag.
+*/}}
+{{- define "splunk-otel-collector.gatewayEnabled" -}}
+{{- or .Values.gateway.enabled .Values.networkExplorer.enabled }}
```

Comment:
Please use `splunk-otel-collector.gateway` helper instead `.Values.gateway` to preserve current behavior in all the places where `splunk-otel-collector.gatewayEnabled` is used